contracts/hackatom/src/contract.rs

```diff
@@ -142,7 +142,7 @@ fn query_verifier<S: Storage, A: Api>(deps: &Extern<S, A>) -> Result<Vec<u8>> {
     // these will be base64 encoded into the json we return, and parsed on the way out.
     // maybe we should wrap this in a struct then json encode it into a vec?
     // other ideas?
-    Ok(addr.as_str().as_bytes().to_vec())
+    Ok(Binary(addr.as_str().as_bytes().to_vec()))
```

Comment:
But why not use `Ok(QueryResponse(addr.as_str()...` here?

Reply:
Because Rust says _can't use a type alias as a constructor_ :(